deploy.md

```diff
@@ -48,9 +48,10 @@ module = config.wsgi:application
 master = true
 processes = 2
 
-socket = /tmp/yunity-dev.sock
+socket = /tmp/fstool.sock
+http-socket = /tmp/fstool.http.sock
```

Comment:
I think I used that so I could run curl commands against it locally on the server to check it responds ok. The non-http socket needs to be requested with uwsgi protocol and command line clients are less/not available.